test/reporter.spec.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable indent */
```

Comment:
Let's only ignore the part where we have to break the indentation rule, not the whole file